standards-catalogue/config.rb

```diff
@@ -90,6 +90,14 @@ def display_status(type, id)
     end
   end
 
+  def display_licence(id)
+    if data.licences[id].url
+      "<td nowrap><strong>#{data.licences[id].type} - <a href=#{data.licences[id].url}>#{data.licences[id].name}</a></strong></td>"
+    else
+      "<td nowrap><strong>#{data.licences[id].name}</strong></td>"
+    end
+  end
```

Comment:
To remove duplicated lookups (not necessarily costly complexity wise, but also has the risk of us leading to us likely forgetting to update a reference in the future)

```suggestion
    license = data.licences[id]
    if license.url
      "<td nowrap><strong>#{license.type} - <a href=#{license.url}>#{license.name}</a></strong></td>"
    else
      "<td nowrap><strong>#{license.name}</strong></td>"
    end
  end
```